bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -261,63 +263,30 @@ public synchronized void login() {
         cloudFallback = false;
 
         try {
-            String urlParameters = "";
+            lastLoginTimestamp = Instant.now();
 
             // if cozytouch, must use oauth server
             if (thingConfig.getCloudPortal().equalsIgnoreCase(COZYTOUCH_PORTAL)) {
-                logger.debug("CozyTouch Oauth2 authentication flow");
-                urlParameters = "jwt=" + loginCozytouch();
+                loginCozyTouch();
```

Comment:
I see a difference with before. In case the code from loginCozyTouch fails, you will now register events while not before.
Maybe the method should return a boolean ?

Reply:
you are right, now the method returns a boolean. thanks